.gitignore

```diff
@@ -7,5 +7,7 @@ payment-reconciler*
 # Lint files
 lint.txt
 
+.DS_Store
 
-.DS_Store 
+# Test coverage file
+coverage.out
```

Comment:
This isn't specified in the styleguides. Should the styleguides be updated?

Reply:
Removed as this file was produced from a flag which has also been removed